src/home/room_screen.rs

```diff
@@ -2147,6 +2200,19 @@ impl RoomScreenRef {
     }
 }
 
+/// Actions for the room screen's tooltip 
+#[derive(Clone, Debug, DefaultNone)]
+pub enum RoomScreenTooltipActions {
+    // Mouse over event when the mouse is over the reaction button
+    // First parameter is rect containing tooltip position and its size
+    // Todo! implement tooltip resizing
+    // The second parameter is tooltip text
+    // The third parameter is tooltip width
+    HoverIn(Rect, String, f64),
```

Comment:
use a structured enum variant for this, like so:
```suggestion
    /// The mouse is hovering over the ??? (see below)
    HoverIn {
        tooltip_pos: Rect,
        tooltip_text: String,
        tooltip_width: f64,
```

Two questions:
1. you wrote "over the reaction button", but I assume you meant "over the row of read receipt Avatars"?
2. If you already have the `Rect`, why do you need a separate `width` value for the tooltip?